uw-frame-components/staticFeeds/notifications.json

```diff
@@ -1,5 +1,17 @@
 {"notifications" :
     [
-      
+      {
+        "id"     : 1,
+        "groups" : ["Everyone"],
+        "title"  : "This is an admin notification smoke test.",
+        "actionURL" : "http://www.google.com",
+        "actionAlt" : "Google",
+        "dismissable" : false,
+        "priority" : true,
+		"buttonText" : [
+		  {"caption" : "Feedback", "url" : "/exclusive/feedback"},
+		  {"caption" : "Return To Old", "url" : "my.wisconsin.edu"}
+		]
+      }
```

Comment:
Accidental? We do have the `sample_notifications.json` file in this same folder.